---
layout: page
title:  "Lesson 10: Spatially variable genes"
order: 10
---

# Lecture

## Learning objectives

By the end of this lesson, we should be able to understand how to identify spatially variable genes in our data using Moran's I. We will also be able to use interactive data visualizations to get some sense for the quality of our analysis.

---

# Hands-on Component

Our in-class hands-on component will visualize our spatial transcriptomics dataset using `gganimate`

---

# Class Notes

Prof. Fan’s notes from class: [genomic-data-visualization-Lesson_10.pptx (click to download)]({{ site.baseurl }}/resources/notes/genomic-data-visualization-Lesson_10.pptx)

Prof. Fan’s code from class: [code-lesson-10.R (click to download)]({{ site.baseurl }}/resources/code/code-lesson-10.R)

---

# Homework Assignment

`HW_5` apply same approach for analyzing your original spatial transcriptomics dataset to the other team's dataset. Your goal is to identify same cell-type (due Saturday evening). See slides for more details on what is expected: [genomic-data-visualization-HW_5.pptx (click to download)]({{ site.baseurl }}/resources/hw/genomic-data-visualization-HW_5.pptx)

`HW_EC1` Make a plot using `gganimate` for extra credit. See slides for more details on what is expected: [genomic-data-visualization-HW_EC1.pptx (click to download)]({{ site.baseurl }}/resources/hw/genomic-data-visualization-HW_EC1.pptx)

---

# Additional resources
- https://gganimate.com/
- https://jef.works/blog/2021/08/12/story-telling-with-data-visualization/
- https://jef.works/blog/2020/12/28/animating-the-cell-cycle/

